### PR TITLE
[CPU] fix set up config for bin conv fused

### PR DESCRIPTION
**Pull request by mandrono**
Issue: 30685

### CODE REVIEW

**Comment by mandrono** [comment]
@dmitry-gorokhov take a look please

File: inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/num_inputs_fusing_bin_conv.cpp [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    binFilterNodeShape.insert(binFilterNodeShape.end(), binConvKernelSize.begin(), binConvKernelSize.end());
+    auto binFilterNode = std::make_shared<ngraph::op::Constant>(ngraph::element::u1, binFilterNodeShape);
+    auto binConv = std::make_shared<ngraph::opset1::BinaryConvolution>(params[0], binFilterNode, strides, padBegins, padEnds, dilations,
+                                                            ngraph::opset1::BinaryConvolution::BinaryConvolutionMode::XNOR_POPCOUNT, padValue, paddingType);
+
+    std::vector<size_t> convFilterNodeShape = {(numOutChannels / numGroups), (binConv->output(0).get_shape()[1] / numGroups)};
+    convFilterNodeShape.insert(convFilterNodeShape.end(), convKernelSize.begin(), convKernelSize.end());
+    auto convFilterNode = std::make_shared<ngraph::op::Constant>(ngraph::element::f32, convFilterNodeShape);
+    auto biasNode = std::make_shared<ngraph::op::Constant>(ngraph::element::f32, std::vector<size_t>{});
+    auto conv = std::make_shared<ngraph::op::ConvolutionIE>(binConv, convFilterNode, biasNode, strides, dilations, padBegins, padEnds, numGroups);
```
Comment (dmitry-gorokhov) [inline inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/num_inputs_fusing_bin_conv.cpp:64]: Why not GroupConvolution from opset1?

File: inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/num_inputs_fusing_bin_conv.cpp [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    const std::vector<ptrdiff_t> padEnds = {0, 0};
+    const std::vector<size_t > dilations = {1, 1};
+    ngraph::op::PadType paddingType = ngraph::op::PadType::EXPLICIT;
+    float padValue = 1.0;
+
+    auto params = ngraph::builder::makeParams(ngraph::element::f32, {inputShapes});
+    std::vector<size_t> binFilterNodeShape = {numOutChannels, inputShapes[1]};
+    binFilterNodeShape.insert(binFilterNodeShape.end(), binConvKernelSize.begin(), binConvKernelSize.end());
+    auto binFilterNode = std::make_shared<ngraph::op::Constant>(ngraph::element::u1, binFilterNodeShape);
+    auto binConv = std::make_shared<ngraph::opset1::BinaryConvolution>(params[0], binFilterNode, strides, padBegins, padEnds, dilations,
```
Comment (dmitry-gorokhov) [inline inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/num_inputs_fusing_bin_conv.cpp:57]: Could you please make a builder and put it inside builders.hpp?

File: inference-engine/tests/functional/plugin/cpu/shared_tests_instances/execution_graph_tests/num_inputs_fusing_bin_conv.cpp [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+#include <vector>
+
+#include "execution_graph_tests/num_inputs_fusing_bin_conv.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+INSTANTIATE_TEST_CASE_P(inputsNumFusingBinConv, ExecGraphOneInputFusingBinConv,
+                        ::testing::Values(fusingBinConvParams{{ 1, 16, 30, 30}, {2, 2}, {3, 3}, 16, 4, CommonTestUtils::DEVICE_CPU}),
```
Comment (dmitry-gorokhov) [inline inference-engine/tests/functional/plugin/cpu/shared_tests_instances/execution_graph_tests/num_inputs_fusing_bin_conv.cpp:13]: Only Depthwise convolutions can be fused into BinConv, It looks like you are testing just grouped conv.

**Pull request merged**